pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -47,6 +48,7 @@ var operatorVersion string
 func init() {
 	operatorNamespace = os.Getenv("OPERATOR_NAMESPACE")
 	operatorVersion = os.Getenv("OPERATOR_VERSION")
+	operatorVersionLabel := k8s.StringToLabel(operatorVersion)
```

Comment:
This is defined in the scope of the init() method. Please define a new var, as done for the other two